flow/genericactors.actor.h

```diff
@@ -209,6 +209,20 @@ Future<T> timeoutError( Future<T> what, double time, int taskID = TaskDefaultDel
 	}
 }
 
+ACTOR template <class T>
+Future<T> timeoutError( Future<ErrorOr<T>> what, double time, int taskID = TaskDefaultDelay ) {
+	Future<Void> end = delay( time, taskID );
```

Comment:
```suggestion
	return timeoutError(throwErrorOr(what), time, taskID);
```
Then you can remove the `ACTOR` and the whole thing is a one-liner.

Reply:
Removed this as part of timeout change and I no longer need this helper.